### ISSUE TITLE
Path returned to LSP client with encoded / char

### ISSUE THREAD

**Issue by rkeithhill** [PowerShell/PowerShellEditorServices#990]
Windows 10 1903 - VSCode Insiders 1.36.0 - PowerShell Preview 2019.5.0

While working on the `posh-git` project I got an error that VSCode could find a file with the path `src%2FConsoleMode.ps1`.  Sure enough the log indicates that it is getting returned by publishDiagnostics with this `/` char encoded:
```
2019-07-01 22:11:09.962 [VERBOSE] tid:20 in 'WriteMessageAsync' C:\PowerShellEditorServices\src\PowerShellEditorServices.Protocol\MessageProtocol\MessageWriter.cs: line 88
    Writing Event 'textDocument/publishDiagnostics'
    
    {
      "jsonrpc": "2.0",
      "method": "textDocument/publishDiagnostics",
      "params": {
        "uri": "file:///c%3A/Users/Keith/GitHub/dahlbyk/posh-git/src%2FPoshGitTypes.ps1",
        "diagnostics": []
      }
    }
```
Hopefully this is an easy fix.

**Comment by rkeithhill** [comment]
Of course, now I can't repro it.  Sigh...  I'll keep at it.  Somehow that `%2F` got it the response msg.

**Comment by TylerLeonhardt** [comment]
This will be fixed with the Omnisharp lib work because they handle Uri's for us

**Comment by andyleejordan** [comment]
Believed resolved — paths now flow through OmniSharp `DocumentUri`, echoing the client URI. Closing as completed; reopen if an encoded `/` still breaks resolution.

---
_Drafted by GitHub Copilot (Claude Opus 4.8) at @andyleejordan's request._
